### DISCUSSION TITLE
Google Device Access "Something Went Wrong"

### DISCUSSION THREAD

**Discussion by ZombieRoxtar** [koush/scrypted#1180] [Q&A]
When I select Login from the Google Device Access plugin, it always leads to "Something Went Wrong". This only started happening in October of 2023. I even tried removing the plugin and creating a brand new project, but it kept on happening.
Docker, 0.57.0
![image](https://github.com/koush/scrypted/assets/10099033/f9fcac36-9715-489a-8552-fbfbf6a190fc)

**Comment by MooreOfNick** [comment]
Yup, I am walking through the set up for these cameras and am running into the same issue.

**Comment by ZombieRoxtar** [comment]
Why is this now a discussion and not a bug when it suddenly stopped working?

**Comment by travelswagger** [comment]
Any update here? I'm still experiencing the same "Something went wrong" problem as of 2024 Jan 6. 

I set OAuth consent screen > Publishing status > In production and still encounter the error. @ZombieRoxtar @MooreOfNick

**Reply by ZombieRoxtar** [reply]
Nope. The whole reason I ran Scrypted was to get a RTSP stream of my Nest doorbell for self-hosted surveillance, but it wasn't totally reliable even when it did work, so I deleted Scrypted from my docker setup. If it ever recorded more than the mailman then I'd hang a cheap IP camera nearby, and I still might,

**Reply by travelswagger** [reply]
Thank you, @ZombieRoxtar. What do you use for a doorbell now? Did you ditch Nest, try Homebridge with Nest, or something else? I hate to rid of my Nest Doorbell, but I guess the front plastic is already showing peeling so it's under consideration. Thanks!

**Reply by ZombieRoxtar** [reply]
Kept Nest, ditched Scrypted.
Now I don't have any recordings and will be at the mercy of Google if anything ever happens in my yard.
Luckily, I live in a quiet area.

**Reply by travelswagger** [reply]
Thanks for that. I can't get over the price increases for Nest Protect, so I'm SOL with Google Nest then. I'm going to give it a try on Homebridge, but I don't have high hopes. I'll keep checking if Nest / Google Device Access ever gets fixed and update here!
